docs/src/types/vectors.md

```diff
@@ -0,0 +1,171 @@
+# Vectors
+
+## Passing in vectors
+
+You can pass a Rust `std::vec::Vec` into your contract method transparently. The following code calls a Sway function which accepts and then returns a `Vec<SomeStruct<u32>>`.
+
+```rust,ignore
+{{#include ../../../packages/fuels/tests/harness.rs:passing_in_vec}}
+```
+
+You can use a vector just like you would use any other type -- i.e. a `[Vec<u32>; 2]` or a `SomeStruct<Vec<Bits256>>` etc.
```

Comment:
```suggestion
You can use a vector just like you would use any other type -- e.g. a `[Vec<u32>; 2]` or a `SomeStruct<Vec<Bits256>>` etc.
```
Maybe more appropriate since it's an example.